Sources/SwifterSwift/CryptoKit/DigestExtensions.swift

```diff
@@ -0,0 +1,17 @@
+// DigestExtensions.swift - Copyright 2022 SwifterSwift
+
+#if canImport(CryptoKit)
+import CryptoKit
+
+@available(iOS 13.0, macOS 10.15, watchOS 6.0, tvOS 13.0, *)
+public extension Digest {
+
+    // MARK: - Properties
+
+    /// SwifterSwift: Hexadecimal value string (read-only)
+    var hexString: String {
+        let bytes: [UInt8] = Array(makeIterator())
```

Comment:
This implementation does one loop in `self` iterator to construct the array, another to construct data with bytes(which could be dropped since we already have the bytes why construct a data with bytes to iterate over those same underlying bytes?), another loop for map and last one for joined. I would suggest we go with a simpler single loop approach. WDYT?
```swift
extension Digest {
  var hexString: String {
    var result = ""
    for byte in self {
      result += String(format: "%02X", byte)
    }
    return result
 }
}
```

Reply:
Changed as suggested with 42e2959a4b2b11b05b26f18ad12b10da4a734faa